### DISCUSSION TITLE
supported audio formats list

### DISCUSSION THREAD

**Discussion by itzaby2k** [deepgram/community#519] [General help]
Are ds2 and  dss audio formats supported in deepgram API

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment]
Hey @itzaby2k, I haven't heard of a ds2 or dss audio file format. The best thing to do is simply make a request with the audio file you'd like transcribed. We give new users many hours of free credits so it won't cost you anything to test out!

**Reply by itzaby2k** [reply]
Thanks for the update. we have medical transcription files with DSS extension, so I checked for the support for dss audio files in Deepgram API, I  already sent a sample file and I got the JSON response as Bad Request: failed to process audio: corrupt or unsupported data (Status: 400)
